parsec/backend/memory/user.py

```diff
@@ -187,11 +185,13 @@ def _get_user_devices(
             results = users.keys()
 
         if omit_revoked:
-            results = [
-                user_id
-                for user_id in results
-                if not user_is_revoked(org._devices[user_id].values())
-            ]
+            now = pendulum.now()
+
+            def _user_is_revoked(user_id):
+                revoked_on = org._users[user_id].revoked_on
+                return revoked_on is not None and revoked_on <= now
```

Comment:
`revoked_on <= now` : so a device can be revoked in the future?

Reply:
in theory yes, but it's not used for the moment (so the check is just to keep things logical)